spyder/utils/syntaxhighlighters.py

```diff
@@ -551,13 +551,17 @@ def rehighlight(self):
 #==============================================================================
 # Cython syntax highlighter
 #==============================================================================
-C_TYPES = 'bool char double enum float int long mutable short signed struct unsigned void'
+C_TYPES = 'bool char double enum float int long mutable short signed struct unsigned void NULL'
 
 class CythonSH(PythonSH):
     """Cython Syntax Highlighter"""
-    ADDITIONAL_KEYWORDS = ["cdef", "ctypedef", "cpdef", "inline", "cimport",
-                           "DEF"]
-    ADDITIONAL_BUILTINS = C_TYPES.split()
+    ADDITIONAL_KEYWORDS = [
+        "cdef", "ctypedef", "cpdef", "inline", "cimport", "extern",
+	"include", "begin", "end", "by", "gil", "nogil", "const", "public",
+	"readonly", "fused ", "static", "api", "DEF", "IF", "ELIF", "ELSE"]

```

Comment:
`"fused "` remove trailing space (this keyword isn't highlighted due the trailing space)